cdap-docs/examples-manual/source/examples/log-analysis.rst

```diff
@@ -0,0 +1,189 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :description: Cask Data Application Platform SparkPageRank Application
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+.. _examples-spark-page-rank:
+
+=======================
+Log Analysis Example
+=======================
+
+A Cask Data Application Platform (CDAP) example demonstrating Spark and MapReduce running in parallel inside a
+Workflow through fork.
+
+Overview
+========
+
+This example demonstrates Spark and MapReduce performing log analysis, computing total number of hits for every unique url, total number of responses for every unique response code and total number of requests made by every unique ip address based on Apache usage log.
+
+Logs are sent to the CDAP and ingested into the *logStream*, which stores the log information event in its entirety.
+
+After these events are streamed, they are taken up by the *ResponseCounterSpark*, which
+goes through the entries, calculates the total number of responses for every unique response code and tabulates results in an KeyValueTable dataset, *responseCount*.
```

Comment:
Add "Oxford" commas: in a list (such as "a, b, and c"), add a comma before "and".
```
...code, and tabulates...